src/styles/blocks/calendar.scss

```
@@ -26,7 +26,7 @@
 
   @for $month-day from 28 through 31 {
     &--month-lenght_#{$month-day} {
-      :nth-child(n + #{$month-day - 1}) {
+      :nth-child(n + #{$month-day +3}) {
```

щось тут не так, мало б працювати так
```suggestion
      :nth-child(n + #{$month-day + 1}) {
```

---

Поправив